src/posts/christmas-change-freeze-2021.md

```diff
@@ -5,13 +5,13 @@ summary: Its that time of the year again when we start to think about changes an
 author: Thomas
 ---
 
-Again it's that time of year when we update you on our change freeze.
```

Comment:
This one wasn't grammarly, it just sounded wrong to me. Tell me if you want to revert